pkg/subnet/local.go

```diff
@@ -807,10 +823,10 @@ func SetDefaultSnapshot(snapshotsDir string, resetCurrentSnapshot bool, avagoVer
 		if err != nil {
 			ux.Logger.PrintToUser("Warning: failure verifying that the local snapshot is the latest one: %s", err)
 		} else if gotSum != expectedSum {
-			downloadSnapshot = true
+			checksumError = true
 		}
 	}
-	if downloadSnapshot {
+	if missingArchive || (checksumError && installUpdates) {
```

Comment:
should it be `missingArchive || checksumError || installUpdates`? if I understand logic correctly

Reply:
it is ok as it is. if the archive is not found, will install. if there is a new update (checksum error) and update install is enabled, will install.